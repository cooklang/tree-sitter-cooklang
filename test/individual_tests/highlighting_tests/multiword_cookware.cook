Use #cast iron skillet{} and #baking sheet{2}.