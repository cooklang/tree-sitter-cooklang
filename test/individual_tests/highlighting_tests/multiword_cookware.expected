(recipe
  (step
    (plain_text)
    (cookware
      name: (cookware_text))
    (plain_text)
    (cookware
      name: (cookware_text)
      (amount
        (quantity)))))